lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/strategic-app-triage.yaml

```diff
@@ -231,3 +231,26 @@ nestedJourneyStates:
     response:
       type: page
       pageId: check-mobile-app-result
+    events:
```

Comment:
We'll want to maintain the existing behaviour from DCMAW CRI - could you compare against these https://github.com/govuk-one-login/ipv-core-back/blob/c8a71c0782e7d5729a3317e1b3b6d5081fb2a98b/lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/app-doc-check.yaml#L13 I think we'll want eg. `fail-with-no-ci` to go to `anotherWay` (aka multiple doc check page)?